articles/internet-protocols/index.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/internet-protocols/
+title: Understanding IPV4 and IPV6 protocols
+description: This article will be a walk through the IPv4 and IPv6 Internet protocols which make communication on the internet possible. 
+author: mackrine-awino
+date: 2021-05-05T00:00:00-13:00
+topics: [Protocols]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/internet-protocols/hero.jpg
+    alt: Internet Protocols Hero Image
+---
+Internet protocols are what make communication on the internet possible. Every device connected to the internet is assigned an IP address. This enables the identification and location of networked computers on the internet. 
+<!--more-->
+Internet protocols also enable the transportation of data items on the internet. Datagrams are transmitted from one device to another without connections to one another by using internet protocols. The datagrams are transmitted with utmost independence from all others.
+
+There are two versions of internet protocol, i.e., Internet Protocol version 4 and internet protocol version 6, used at the network layer of the [OSI model](https://www.imperva.com/learn/application-security/osi-model/). 
+
+### Table of contents
+
+1. [Prerequisites](#prerequisites)
+2. [Introduction to IPv4 and IPv6](#introduction-to-IPv4-and-IPv6)
+3. [Internet protocol version 4](#internet-protocol-version-4)
+4. [Internet protocol version 6](#internet-protocol-version-6)
+5. [Similarities between IPv4 and IPv6](#similarities-between-IPv4-and-IPv6)
+6. [Differences between IPv4 and IPv6](#differences-between-IPv4-and-IPv6)
+7. [Why does IPv6 take preference to IPv4?](#why-does-IPv6-take-preference-to-IPv4)
+8. [Conclusion](#conclusion)
+
+### Prerequisites
+
+To understand the content of this article, a reader should have the following:
+- A beginner's understanding of computer networking.
+
+- Basic knowledge of internet communication and data transmission.
+
+
+### Introduction to IPv4 and IPv6
+
+IPv4 and IPv6 are the actual protocols tasked with data transmission in the form of packets and datagrams. 
+
+IPv4 protocol is mainly used with [ethernets](https://en.wikipedia.org/wiki/Ethernet)	during packet switching in the link-layer networks.	
+
+IPv6 protocol being more newfangled has improved capabilities as compared to IPV4. It can even have infinite addresses hence is replacing IPV4 due to the daily growth in technology increase number of networks being set up frequently.
```

Comment:
```suggestion
IPv6 protocol being more newfangled has improved capabilities as compared to IPV4. Hence, it can even have infinite addresses replacing IPV4 due to the daily growth in technology and increases the number of networks being set up frequently.
```